### PR TITLE
Feature/135 changing cauli module to cauliframework

### LINKED ISSUE #135: Rename the module from Cauli to something different to be able to use it as a namespace
Currently the framework is called `Cauli` as well as one class within the framework. This makes the module-namespace unusable. If one for example wants to use `Cauli.Configuration` the swift compiler interpretes `Cauli` as the class and thus it will not compile.

My proposal ist to rename the framework and module to `Cauliframework`.

### PR DESCRIPTION
**Pull request by brototyp**
This PR closes #135.

This PR is a little overwhelming because of all those change in the Project, Workspace and CocoaPod related files. I will try to add some pointers in the diff to ease the review.

### CODE REVIEW

File: Cauli/Cauli.swift [hunk 1]
```diff
@@ -20,7 +20,7 @@
 //  THE SOFTWARE.
 //
 
-import Foundation
+import UIKit
```
Comment (brototyp) [inline Cauli/Cauli.swift:23]: These are theoretically unrelated to this here, but I noticed it anyways and fixed it.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -29,5 +29,5 @@ Somebody will add two eyes to your code and might give some feedback. Got some c
 Checked out the repository and would like some guidance to find your way?
 
 * We are using Cocoapods to manage our two testing dependencies, [Quick](https://github.com/Quick/Quick) and [Nimble](https://github.com/Quick/Nimble).
-* Just open the `Cauli.xcworkspace`. Here you can find all the code that make up Cauli.
+* Just open the `Cauliframework.xcworkspace`. Here you can find all the code that make up Cauli.
```
Comment (brototyp) [inline CONTRIBUTING.md:32]: Since the .xcodeproj changed it changed the .xcworkspace file as well.

**Review by brototyp — COMMENTED** [review]
There are barely any really interesting changes here. Most are just replacing `Cauli` with `Cauliframework`.

**Comment by pstued** [comment]
I think in this PR we should also change the bundle identifier from `de.brototyp.Cauli` to `works.cauli.cauliframework`. What do you think?

**Comment by pstued** [comment]
Don't we also have to adjust the module in the `.jazzy` file?

**Comment by brototyp** [comment]
Both are great points I haven’t thought about! Thanks! I changed both of them.

**Review by pstued — APPROVED** [review]

**Pull request merged**